packages/core-react/src/walletData/walletConnectMobileLinks.ts

```diff
@@ -0,0 +1,45 @@
+export const walletConnectMobileLinks = [
```

Comment:
do we have some type to validate this names? Like, `WalletName[]` or `Array<Wallet['name']>`

Reply:
<img width="540" alt="image" src="https://user-images.githubusercontent.com/1245209/235898718-744c5a3e-1e3d-451e-83ae-b0eb710f623e.png">